app/services/criteria_matcher.py

```
@@ -361,7 +361,7 @@ def _get_macos_last_open_time(file_path: Path) -> Optional[float]:
 
             # Method 2: Try xattr (fallback)
             xattr_result = subprocess.run(
-                ["xattr", "-p", "com.apple.lastuseddate#PS", str(file_path)],
+                ["xattr", "-p", "com.apple.lastuseddate#PS", "--", str(file_path)],
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)